shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -23,8 +23,8 @@
             <description>When enabled, the motors will spin at the ChannelNeutral command when armed with zero throttle.</description>
         </field>
 
-        <field name="MotorInputOutputCurveFit" units="-" type="float" elementnames="A,B" defaultvalue="1,1">
-            <description>Actuator mapping of input in [-1,1] to output on [-1,1], using power equation of type a*x^b.</description>
+        <field name="MotorInputOutputCurveFit" units="-" type="float" elements="1" defaultvalue="1">
```

Comment:
Yes, that's fair.

I can't promise smaller, more atomic PRs-- it would hard for me to justify serializing two related changes in this module (or especially commits, at this point-- we're having some degree of internal debate about squashing-down most PRs into individual commits).  But I'll endeavor to note things like this in the comments.

And I'd be lying if efficiency wasn't a small concern- when I'm cleaning up performance for F1, if I see something that I have always thought was vestigial, I'm inclined to take it away.  But one multiply does not make or break this PR.  Odds are that even the change to the actuator half is optional.

P.S. It's possible that the a term could be actually useful, if one desired to deliberately clip or cut off part of the ESC range.  I would think this would belong in output settings though (or in the case of the ESC, setting a smaller maximum thrust/RPM target to correspond to '1.00' == full effort.